### PR TITLE
[enhancement](sequence col) add session variable to skip sequence column check while INSERT INTO

### PR DESCRIPTION
**Pull request by zhannngchen**
## Proposed changes

Issue Number: close #xxx

For unique key tables using a sequence column, the `INSERT INTO` operation requires a sequence value to be provided for each row. 
This PR add a new session variable `require_sequence_in_insert` to control this behavior.

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java [hunk 1]
```diff
@@ -2177,6 +2178,13 @@ public void setIgnoreShapePlanNodes(String ignoreShapePlanNodes) {
     })
     public boolean enablePhraseQuerySequentialOpt = true;
 
+    @VariableMgr.VarAttr(name = SKIP_INSERT_SEQUENCE_CHECK, needForward = true, description = {
+            "对于使用了sequence列的unique key表，insert into操作会要求必须提供每一行的sequence列的值，开启该变量后将跳过该检查",
+            "For unique key tables using a sequence column, the INSERT INTO operation requires a sequence value"
+                    + " to be provided for each row. When this variable is enabled, the check will be skipped"
+    })
+    public boolean skipInsertSequenceCheck = false;
```
Comment (starocean999) [inline fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java:2186]: better to use enableInsertSequenceCheck instead

**Review by gavinchou — DISMISSED** [review]

**Comment by zhannngchen** [comment]
run buildall

**Comment by zhannngchen** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by Yukang-Lian — APPROVED** [review]

**Pull request merged**